docs/istio.md

```diff
@@ -0,0 +1,71 @@
+# Istio Usage in ARO HCP
+
```

Comment:
Diagram or link to pertinent diagram will help reader put the text that follows into context